lib/fluent/plugin/out_color_stripper.rb

```diff
@@ -8,8 +8,6 @@ class ColorStripperOutput < Output
     def configure(conf)
       super
 
-      @tag = conf.fetch('tag') { raise ArgumentError, 'tag field is required to direct transformed logs to' }
```

Comment:
> Or not? 

Fluentd checks mandatory values are present or absent on boot.
If mandatory values are absent in fluent.conf then Fluentd raises `Fluent::ConfigError` and print logs like:

```
[error]: config error file="out.conf" error_class=Fluent::ConfigError error="'tag' parameter is required"
```

I believe this is useful error message for users.